webapp/src/plugin_hooks.ts

```diff
@@ -55,7 +50,8 @@ export default class Hooks {
         if (!isUserConnected(this.store.getState())) {
             await this.store.dispatch(getConnected());
             if (!isUserConnected(this.store.getState())) {
-                this.store.dispatch(sendEphemeralPost('Your Mattermost account is not connected.'));
+                const providerConfiguration = await getProviderConfiguration(this.store.getState());
+                this.store.dispatch(sendEphemeralPost(`Your Mattermost account is not connected to ${providerConfiguration.DisplayName}. In order to create a calendar event please connect your account first using \`/${providerConfiguration.CommandTrigger} connect\`.`));
```

Comment:
We typically use snake case for the JSON serialization of things, but 0/5 here

Reply:
I didn't bother since this wasn't intended to be public, but I can change it.